model/jvm/metrics-experimental.yaml

```diff
@@ -67,3 +67,11 @@ groups:
     brief: "Number of buffers in the pool."
     instrument: updowncounter
     unit: "{buffer}"
+
+  - id: metric.jvm.file_descriptor.count
+    type: metric
+    metric_name: jvm.file_descriptor.count
+    stability: development
+    brief: "Number of open file descriptors by the JVM."
```

Comment:
can you include a notes section that describes what API(s) we would typically get this data from? I think you can see examples of this in the other JVM metrics